tools/coffee-manager/org/contikios/coffee/CoffeeManager.java

```diff
@@ -151,15 +151,12 @@ public static void printStatistics(CoffeeFS coffeeFS) {
 		CoffeeConfiguration conf = coffeeFS.getConfiguration();
 
 		try {
-			Iterator<Map.Entry<String, CoffeeFile>> iterator =
-				coffeeFS.getFiles().entrySet().iterator();
-			while (iterator.hasNext()) {
-				Map.Entry<String, CoffeeFile> pair = iterator.next();
-				CoffeeFile file = pair.getValue();
-				bytesWritten += file.getLength();
-				bytesReserved += file.getHeader().getReservedSize();
-				fileCount++;
-			}
+      for (var pair : coffeeFS.getFiles().entrySet()) {
+        CoffeeFile file = pair.getValue();
+        bytesWritten += file.getLength();
+        bytesReserved += file.getHeader().getReservedSize();
+        fileCount++;
+      }
```

Comment:
Nice code cleanup but since the key is not used, you could make it even simpler by using `coffeeFS.getFiles().values()` instead. Same for other loops in this file.

```
      for (CoffeeFile file: coffeeFS.getFiles().values()) {
        bytesWritten += file.getLength();
        bytesReserved += file.getHeader().getReservedSize();
        fileCount++;
      }
```

Reply:
Fixed.